src/components/Dropzone/Dropzone.mdx

```diff
@@ -0,0 +1,66 @@
+import {
+  Preview,
+  Story,
+  ArgsTable,
+  PRIMARY_STORY,
+} from "@storybook/addon-docs";
+import Dropzone from "./Dropzone.vue";
+
+# Dropzone
+
+The Dropzone is a form field with an input type=file, styled to look like a drop area.
+
+In addition to a mouse click that opens the select-file dialog, the drop zone can accept files being drag&dropped onto it.
+
+It supports single file selection (or drop), and has built-in checks for the file type and size (based on props).
+
+<Preview>
+  <Story id="components-dropzone--primary" />
+</Preview>
+
+## How to Use
+
+All text shown in the Dropzone must be added as the `textContent` prop (Object).
+
+The required keys for the `textContent` are: `dragAndDropHere`, `yourFile`, `or`, `uploadFileButtonText`,
+`acceptedFormatIs`, `acceptedFormatsAre`, `maxFileSizeIs`, `downloadSampleFile`,
+`uploading`, `mightTakeAMinute`, `looksGreat`, `removeFileButtonText`,
+`couldNotUpload`, `canOnlySelectOneFile`, `fileIsTooLarge`, `fileTypeNotValid`,
+`defaultErrorText`, `errorMessage`, `successMessage`
+
+The prop `uploadId` is also required (so that two or more Dropzones can be added to the same page).
+
+```html
+<Dropzone
+  upload-id="dropzone"
+  accept-type=".csv"
+  max-size-in-bytes="2147483648"
+  :show-type-and-max-size="true"
+  :show-sample-link="false"
+  :status="null"
+  :file-ob="fileObject"
+  :text-content="textContentObject"
+  @select="uploadFile"
+  @remove="removeFile"
+/>
+```
+
+Other supported props (optional):
+
+`acceptType`: takes an array of Strings that are passed into the `accept` file types of the input. (example: '.csv' or '.jgpeg,.jpg,.png')
```

Comment:
It looks like the component is actually just taking a string not an array of strings

Reply:
You could instead word this "comma delimited string list"

Reply:
oh yes indeed 😬